datafusion/sqllogictest/test_files/topk.slt

```diff
@@ -49,7 +49,12 @@ select * from topk order by x desc limit 3;
 8
 5
 
-
+query I
```

Comment:
I verified this test fails without the changes in this PR

```
Completed in 3 seconds                                                                                                                                        External error: query result mismatch:
[SQL] select * from (select * from topk limit 8) order by x limit 3;
[Diff] (-expected|+actual)
    0
    1
-   2
+   2
+   2
+   3
+   3
+   3
+   4
at test_files/topk.slt:52
```

Reply:
exactly